share/ci/n_wise_generator.py

```diff
@@ -127,7 +128,7 @@ def is_valid_combination(row):
             return False
         else:
             # install/clang.sh is currently only providing apt sources up to clang 18
```

Comment:
```suggestion
            # install/clang.sh is currently only providing apt sources up to clang 19
```

Reply:
Better because lower maintenance overhead:
```suggestion
            # install/clang.sh is currently not providing apt sources later than this clang version
```
Proof: See your comment